src/encoding.cc

```diff
@@ -271,3 +271,77 @@ double DecodeDouble(const char *ptr) {
   memcpy(&value, &decoded, sizeof(value));
   return value;
 }
+
+char* EncodeVarint32(char *dst, uint32_t v) {
+  // Operate on characters as unsigneds
+  unsigned char* ptr = reinterpret_cast<unsigned char*>(dst);
+  static const int B = 128;
```

Comment:
I prefer `0b1000'0000` or `0x80` rather than decimal form here.

Reply:
And some other decimal numbers for bit-wise computation below.

Reply:
Yes, I also prefer to `0x80` over 128. We still need to use `ob1000000` if want to use binary representation since the `'` digital separator was introduced in C++14.

Reply:
Refactored this function. Replace it to unrolled version.